.github/workflows/on_PR_windows_matrix.yml

```diff
@@ -156,15 +156,15 @@ jobs:
       SHELLOPTS: igncr
     defaults:
       run:
-        shell: C:\tools\cygwin\bin\bash.exe -eo pipefail '{0}'
+        shell: C:\cygwin\bin\bash.exe -eo pipefail '{0}'
     steps:
       # Make sure we don't check out scripts using Windows CRLF line endings
       - run: git config --global core.autocrlf input
         shell: pwsh
       - uses: actions/checkout@v3
 
       - name: Set up Cygwin
-        uses: egor-tensin/setup-cygwin@v3
+        uses: cygwin/cygwin-install-action@master
```

Comment:
Should we maybe go for a known tag, v3 currently?

Reply:
Yes, I will make the change :)